src/Lumper.UI/ViewModels/MainWindowViewModel.cs

```diff
@@ -34,7 +34,32 @@ public MainWindowViewModel()
                 RxApp.MainThreadScheduler);
         }
     }
+    public async ValueTask UpdateCommand()
+    {
+        bool updateAvailable = await Lumper.UI.Updater.Updater.CheckForUpdate();
+        if (updateAvailable)
+        {
+            ButtonResult result = await MessageBoxManager
+                .GetMessageBoxStandard(
+                    "Update Available",
+                    "Update available. Download now?", ButtonEnum.OkCancel)
```

Comment:
Let's be more verbose: "An update is available (version number), do you want to download and restart?